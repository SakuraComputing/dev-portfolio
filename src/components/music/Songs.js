import React from 'react';

const Songs = (props) => (

    <li>
        <i className="fa fa-headphones"></i>
        <a href="#">
            {props.song.title}
        </a>
    </li>
);
export default Songs;