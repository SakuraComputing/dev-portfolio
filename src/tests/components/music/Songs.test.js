import React from 'react';
import { shallow } from 'enzyme';
import Songs from '../../../components/music/Songs';

describe('Songs', () => {

    let wrapper, song;

    beforeEach(() => {
        song = jest.fn();
        wrapper = shallow(<Songs song={song}/>);

    });
    it('should render the for correctly', () => {
        expect(wrapper).toMatchSnapshot()
    })
});