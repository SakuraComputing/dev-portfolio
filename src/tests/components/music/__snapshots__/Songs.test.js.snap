// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Songs should render the for correctly 1`] = `
<li>
  <i
    className="fa fa-headphones"
  />
  <a
    href="#"
  />
</li>
`;
